themes/gatsby-theme-blog/src/theme/colors.js

```diff
@@ -1,19 +1,19 @@
-const teal = `#66b9bf`
-const darkTeal = `#078092`
-const darkBlue = `#282c35`
-const darkGray = `#222`
-const almostBlack = `#1a1a1a`
+const gatsby = `#663399`
+const gatsbyDarker = `#362066`
+const gatsbyBright = `#D9BAE8`
```

Comment:
NB: I know @fk has some sort of number-based naming convention for .org. Even if we're only using a subset of the brand colors, it might make sense to use the same naming convention

Reply:
yeah i was looking at the draft brand guidelines for this actually! some of them are "named" named and some of them are numbered. down for whatever @fk thinks makes the most sense.

Reply:
TBH I don't have strong opinions on naming other than I think that most of these aliases

https://github.com/gatsbyjs/gatsby/blob/3c47ad1677952d14d9ac23463de3c9ab1c3a18a4/www/src/utils/tokens/colors.js#L119-L125

should go in favor of "role-based tokens" https://github.com/gatsbyjs/gatsby/blob/3c47ad1677952d14d9ac23463de3c9ab1c3a18a4/www/src/utils/tokens/colors.js#L143-L185

But of course having an alias for "Gatsby Purple" makes sense.

---

Sorry, bikeshedding. In regards to the names here I first wanted to say "I'm fine with the names as they are", but thinking about it again, @jxnblk is probably right in keeping things consistent — makes sense as a reference for ourselves. Then again we could solve the latter issue in so many ways. I feel like I'm not adding much here today anymore — gotta sleep, head hurts!

---

Oh @jxnblk — you know I'll take your veto re: `purple.60` any day, right? 😉
`blackFade[80]` already creeps me out: https://github.com/gatsbyjs/gatsby/blob/3c47ad1677952d14d9ac23463de3c9ab1c3a18a4/www/src/utils/tokens/colors.js#L127-L134